lib/mix/lib/mix/utils.ex

```diff
@@ -147,7 +147,9 @@ defmodule Mix.Utils do
 
   The callback will be invoked for each node and it
   must either return `{printed, children}` tuple or
-  `false` if the given node must not be printed.
+  `false` if the given node must not be printed. If
+  `printed` is `nil`, the node will not be printed,
+  but any `children` will.
   """
   @spec print_tree(term, (term -> {String.t, [term]}), Keyword.t) :: :ok
```

Comment:
I think this spec it out of date (it should be a tuple inside the function and also String.Chars.t).